RocketChat/Rocket.Chat.ReactNative
# fix: quoting a image with a custom emoji in description is breaking layout

## Proposed changes

Remove the margins at custom emojis wrapped by <Text> component
## Issue(s)	


## How to test or reproduce

- A user sends an image and in the description, he adds custom emoji
- Then, quote that image+description

- On iOS you will see a visual bug
- If you didn’t see it yet, you can change the screen orientation, and it will appear

## Screenshots

#### Before
https://user-images.githubusercontent.com/47038980/232122066-5cedeb31-fdcb-49c2-8998-01f541c66299.mov

#### After
https://user-images.githubusercontent.com/47038980/232122209-7db74735-f36d-4b7b-996f-5d260cd557f3.mov

## Types of changes




## Checklist



## Further comments

[TC-632]

[TC-632]: https://rocketchat.atlassian.net/browse/TC-632?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
